src/mlpack/core/cv/metrics/topk_accuracy_impl.hpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file topk_accuracy_impl.hpp
+ * @author Arunav Shandeelya
+ * 
+ * The implementation of class topK_accuracy_score
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_IMPL_HPP
+#define MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_IMPL_HPP
+
+namespace mlpack {
+namespace cv {
+    template<typename MLAlgorithm, typename DataType, typename ResponseType, typename TopK>
+    double TopK_Accuracy::Evaluate(MLAlgorithm& model,
+                                    const DataType& data,
+                                    const arma::Row<size_t>& labels,
+                                    const TopK& k)
+
+    {
+        if(data.n_cols! = labels.n_cols)
+        {
+            std::ostringstream fp;
+            fp << "topK_accuracy_score::Evaluate(): number of inputs ("<< data.n_cols <<")" 
+            << "does not matches with number of target labels ("<< labels.n_cols <<")" 
+            <<std::endl;
+        throw std::invalid_argument(fp.str());
+        }
+        arma::Row<size_t> predictedLabels;
+        // Taking Classification output from the model.
+        model.Classify(data, predictedLabels)
+        // Shape of the predicted values.
+        double predicted_class = predictedLabels.size();
+        // Top 'K' label prediction class. 
+        if (k<predicted_class[1]){
+            const int idx = predicted_class[1] - k - 1;
+        }
+        else:
+        {
+            std::ostringstream fs;
+            fs <<"There are less number of class for topK accuracy score" <<std::endl;
+        }
+        const int idx = predicted_class[1] - k - 1;
+        const float c = 0;
```

Comment:
Do you mind using a more descriptive name. Thanks.

Reply:
Also you are adding 1 to it below but this is a const type, which is one of the reasons for the build error.

Reply:
@kartikdutt18  Yeah changed c to 'count', Is this seems good?

Reply:
@kartikdutt18 Okay I got your point, then how can we add 1 over there for update the count by 1 for each loop?

Reply:
Simply remove `const`.

Reply:
>  Is this seems good?
Yes.

Reply:
Okay, I removed the `const` from there. Thanks